turbo.json

```diff
@@ -4,22 +4,23 @@
 		"kit.svelte.dev#build": {
 			"dependsOn": ["^build"],
 			"inputs": ["src/**", "../../packages/kit/docs/**", "../../documentation/**"],
-			"outputs": [".vercel_build_output/**", ".vercel/output/**"],
+			"outputs": [".vercel/output/**"],
 			"outputMode": "new-only",
-			"env": ["VERCEL", "ENABLE_VC_BUILD"]
+			"env": ["VERCEL"]
 		},
-		"build": {
+		"create-svelte#build": {
 			"dependsOn": ["^build"],
 			"inputs": ["src/**", "scripts/**", "shared/**", "templates/**"],
-			"outputs": [
-				"files/**",
-				"dist/**",
-				".svelte-kit/**",
-				".vercel_build_output/**",
-				".vercel/output/**"
-			],
+			"outputs": ["dist/**"],
+			"outputMode": "new-only",
+			"env": ["VERCEL"]
+		},
+		"build": {
```

Comment:
do we need this since the `kit` project isn't built? if so, it could be worth a comment (comments should work in this file because it's parsed with json5 which supports them)